recipes/AdaptiveCpp/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "AdaptiveCpp" %}
+{% set version = "24.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/AdaptiveCpp/AdaptiveCpp/archive/v{{ version }}.tar.gz
+    sha256: 3bcd94eee41adea3ccc58390498ec9fd30e1548af5330a319be8ce3e034a6a0b
+    patches:
+      - patches/fallback-missing-functions.patch  # [linux]
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('adaptivecpp', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - ninja  # [not win]
+  host:
+    - clangdev
+    - libboost-devel
+    - llvm-openmp >=14,<19
+    - llvm >=14,<19
+    - spirv-tools
+  run:
+    - llvm >=14,<19
+
+test:
+  commands:
+    - test -f $PREFIX/lib/cmake/AdaptiveCpp/adaptivecpp-config.cmake  # [not win]
+    - if not exist %PREFIX%\\Library\\lib\\cmake\\AdaptiveCpp\\adaptivecpp-config.cmake exit 1  # [win]
```

Comment:
I suggest that you try to compile a simple test program with these configs in order to check that the paths in these cmake files are correct.

Alternatively, you can manually look to see that they don't contain hardcoded paths to the BUILD_PREFIX. Only paths to the placeholder prefix are valid because those will be replaced at install time with the end user's environment path.